target/linux/x86/patches-6.6/300-CE2600-support.patch

```diff
@@ -0,0 +1,4840 @@
+From 120998ebc605fefccd456df7eb2d3e8199e4d668 Mon Sep 17 00:00:00 2001
```

Comment:
This patch should be split into smaller parts. Did you tried to discuss it upstream?

Reply:
You mean on the linux mailing lists? No, I didn't try to do so. I can split this into multiple patch files where I modify something, and plain files where I've added new files. Please note I've added a driver for the horrible x86-arm bridge, but it doesn't do anything. I should get rid of that.

Reply:
I am just an OpenWrt developer, and this is only my point of view.